### ISSUE TITLE
Sending emails

### ISSUE THREAD

**Issue by JMStudiosJoe** [Opportunity-Hack-San-Jose-2017/Team3#27]
When new volunteer signs up need to send email to webmanager@givelight.org with these details

* Name: | Maram Dawood
-- | --
* Email: | maram.t.dawood@hotmail.com
Phone: | 01099559030
City: | Cairo
* Country: | Egypt
Interests: | Event Planning,Teaching,Public Speaking,Other
Profession/Skillset: | International Business/Finance

**Comment by JMStudiosJoe** [comment]
Will need to talk to Uroosa since more than likely they are using a Joolma plugin for handling their emails. We have the To: email but not the from yet

**Comment by RaghuMurugesan** [comment]
I can help with this. We need access to their `DNS account` for getting a "from" email ID.

**Comment by JMStudiosJoe** [comment]
Yes I have already asked for the from email might need to talk to Uroosa though, we shall see. Should be able to just look at the email sent to them and at that "from" address. Please assign yourself if you want to take on this issue.

**Comment by RaghuMurugesan** [comment]
@JMStudiosJoe 

- Do you have password for  webmanager@givelight.org? 
- We need to get it from the owner of `webmanager@givelight.org` and use this email ID to sign up a for [MailGun Account](https://www.mailgun.com)
- I will use my account's temporary credential for development purpose

**Comment by RaghuMurugesan** [comment]
BELOW NEEDED FROM GIVE LIGHT TEAM FOR MAKING THIS MODULE READY FOR PRODUCTION:
**mailGunUserName
mailGunPassword**

REFER TO THIS LINK FOR MORE INFO
https://documentation.mailgun.com/en/latest/faqs.html

BELOW ARE THE INFO WE HAVE FOR THIS MODULE WHICH WE WILL ADD TO PRODUCTION CREDENTIALS
email_host: "smtp.mailgun.org",
Our App's Sender Email ID(Aka From ID):  webmanager@givelight.org
